jobs/alertmanager/templates/config/alertmanager.yml

```
@@ -51,6 +51,10 @@ global:
   pagerduty_url: <%= url %>
   <% end %>
 
+  <% if_p('jira_api_url') do |jira_url| %>
```

```suggestion
  <% if_p('alertmanager.jira.url') do |jira_url| %>
```

you need to use the spec here that you specified earlier